src/components/App/App.jsx

```diff
@@ -146,8 +177,8 @@ const handleGetRecipeInstructions = (recipeId) => {
           <Routes>
             <Route path="/" element={<Home />}/>
             <Route path="/profile" element={<Profile />}/>
-            <Route path="/recipes" element={<Recipes recipeinfo={recipeinfo} handleRecipeCardClick={handleRecipeCardClick} recipeInstructions={recipeInstructions} handleRecipeModal={handleRecipeModal} recipeModelShow={recipeModelShow} recipes={sampleRecipes}/>}/>
-            <Route path="/pantry" element={<Pantry productForm={productForm} handleOnProductFormChange={handleOnProductFormChange} handleOnSubmitProductForm={handleOnSubmitProductForm} handleProductQuantity={handleProductQuantity} products={products}/>}/>
+            <Route path="/recipes" element={<Recipes recipeinfo={recipeinfo} handleRecipeCardClick={handleRecipeCardClick} recipeInstructions={recipeInstructions} handleRecipeModal={handleRecipeModal} recipeModelShow={recipeModelShow} recipes={recipes}/>}/>
+            <Route path="/pantry" element={<Pantry operations={Operations} productForm={productForm} handleOnProductFormChange={handleOnProductFormChange} handleOnSubmitProductForm={handleOnSubmitProductForm} handleProductQuantity={handleProductQuantity} products={products}/>}/>
```

Comment:
Just curious, what motivated moving Operations into this property?

Reply:
Since I was creating each `"add"` `"subtract"` `"erase"` strings into a `Symbol`, I was creating a unique value for each operation. When I had two separate Operations enums (one in App.jsx and in ProductCard.jsx), even when the operations were the same, I was comparing two different values. I had to pass in the same Operations enum to make sure I was comparing the same values